resources/server/api/ogc/templates/wfs3/header.html

```diff
@@ -13,12 +13,12 @@
     <link rel="stylesheet" href="https://unpkg.com/leaflet.fullscreen@3.0.2/Control.FullScreen.css"
             integrity="sha384-PALxxXbG28eiE9bT7yKmsbjd4Ta3qKMLlzsrp6G+05f4Nltt4ktlg1l7e4RPeNEa"
             crossorigin=""/>
-    <link rel="stylesheet" href="{{ static( "style.css" ) }}" />
+    <link rel="stylesheet" href="{{ static( &quot;style.css&quot; ) }}" />
```

Comment:
Same here, I think these are incorrect

Reply:
Yes, `{{ static( ` and ` ) }}` become strings but the parts between does not. I assume my fix is wrong, but I don't know it should be written instead

Reply:
Yes, {{ static(  and  ) }} become strings but the parts between does not. I assume my fix is wrong, but I don't know how it should be written instead